imports.go

```diff
@@ -87,24 +89,51 @@ func (b Column) Less(other Record) bool {
 	return false
 }
 
-func ColumnCSVUnmarshaller() CSVRecordUnmarshaller {
-	return ColumnCSVUnmarshallerWithTimestamp("")
+type CSVFormat uint
```

Comment:
I agree again. Maybe we should remove csv stuff altogether (maybe move that kind of code to its own conveniences repo).

Reply:
the pdk might be a good place - it already has some code for csv handling

Reply:
Moved the `csv` stuff to its package but then I had to move some stuff to the `imports` package, since `PilosaClient` depends on the csv stuff for exports (and vice versa). That change increased the lines to be reviewed quite a bit, but most of them are renames, so hopefully that would be trivial.

Reply:
I'm a bit confused about the exporter - shouldn't it basically just return an io.Reader so that the user can write the exported data to wherever?

Reply:
I am not sure whether the user would like to write the exported data verbatim, they may transform it or maybe not write it all. Since we keep the CSV stuff in a subpackage, I think it makes sense to use it to make the exported data more useful for the user.

Reply:
One could imagine Pilosa's export functionality being a lot more general than just dumping a CSV. It could support multiple content types - perhaps even in a pluggable way. I think that the responsibility of decoding that data falls outside the go-pilosa package. It also goes against the package structure that we've generally adopted to have the core package depend on subpackages - we've striven to make the reverse true in Pilosa.

Reply:
OK, ditched the imports package, so the core doesn't depend on it, but that requires refactoring the integration tests. Hold on...

Reply:
sorry :/